synapse/lib/cell.py

```diff
@@ -2822,6 +2823,8 @@ def reload():
 
             self.addReloadableSystem('https:certs', reload)
 
+        self.https_listeners.append({'host': lhost, 'port': lport})
```

Comment:
Maybe an optional `name=None` kwarg so the name can be put in this dictionary for a little extra context?

Reply:
what name would that be?

Reply:
I don't know. Some human context that someone can provide when calling `addHttpsPort()`. Something like `await addHttpsPort(1234, name='control')`? Never mind, it's a bad idea.